frontend/public/extend/devconsole/components/import/BuilderImageCard.scss

```
@@ -0,0 +1,39 @@
+.odc-builder-image-selector__card {
+  position: relative;
+  height: 110px;
+  width: 132px;
+  margin: 8px;
+  text-align: center;
+
+  &-icon {
```

```suggestion
  &__icon {
```
Same pattern for all the other element selectors.